### PR TITLE
Jumpand blink update

### PR DESCRIPTION
**Pull request by meinardxd**
pls approve i changed the blink gif but i will still work on the jump gif so you will be satisfied

### CODE REVIEW

File: src/utils/database.js [hunk 1]
```diff
@@ -74,7 +74,7 @@ export const getUtangs = async (setUtangs) => {
       const list = navigator.onLine
         ? records.reverse()
         : JSON.parse(localStorage.getItem("utangs") || []);
-      setUtangs(list);
+      setUtangs([]);
```
Comment (gabcamba) [inline src/utils/database.js:77]: why did you set utang list to empty array? This will cause a bug, users wont see their list

**Review by gabcamba — CHANGES_REQUESTED** [review]

File: src/utils/database.js (hunk 1 above)
Comment (meinardxd) [reply src/utils/database.js:77]: edi sorry

File: src/gifs/Group-Jump-Final.gif
Comment (gabcamba) [inline src/gifs/Group-Jump-Final.gif:1]: why is this file added but not used in code? Please remove if not imported to code @meinardxd

File: src/utils/database.js (hunk 1 above)
Comment (gabcamba) [reply src/utils/database.js]: resolved

File: src/gifs/Group-Jump-Final.gif
Comment (gabcamba) [inline src/gifs/Group-Jump-Final.gif:1]: why is this file added but not used in code? Please remove if not imported to code @meinardxd

**Review by gabcamba — CHANGES_REQUESTED** [review]

File: src/gifs/Group-Jump-Final.gif
Comment (meinardxd) [reply src/gifs/Group-Jump-Final.gif:1]: Removed! 😘

**Review by gabcamba — APPROVED** [review]
lgtm approved
